test/session.test.ts

```diff
@@ -1270,4 +1270,100 @@ describe('MediaSession', () => {
             });
         });
     });
+
+    describe('Custom Attributes', () => {
+        it('should propogate Session attributes created Events', () => {
+            const sessionOptions = {
```

Comment:
super small nit, but I'd suggest changing the variable name to match our comments in line in `src/session.ts`
```suggestion
            const sessionAttributes = {
```